modules/component/src/componentNext.ts

```diff
@@ -251,7 +314,41 @@ export class ComponentNext<P1 extends ComponentProps> {
       this._comparator
     )
   }
-
+  /**
+   * Adds presentation logic to the component
+   * 1. Create view based on props and state
+   *
+   * ```typescript
+   * import {ComponentNext} from '@action-land/component'
+   *
+   * const component1 = ComponentNext.lift(10).render((_, props: string) => [props, _.state + 1])
+   * component._view({}, component._init(), 'Hello') // output: [Hello, 11]
+   * ```
+   * 2. Can invoke child component's view
+   *
+   * ```typescript
+   * import {ComponentNext} from '@action-land/component'
+   *
+   * const component = ComponentNext.lift('Hello').install({
+   *   child: ComponentNext.lift('World').render((_, p: string) => p)
+   * })
+   * .render((_, p: string) => [p, _.children.child('World')]
+   * component._view({}, component._init(), 'Hello') // output: [Hello, World]
+   * ```
+   * 3. Can emit action
+   *
+   * ```typescript
+   * import {ComponentNext} from '@action-land/component'
+   *
+   * const component = ComponentNext.lift(10)
+   * .matchR('add', (a: number, s) => s + a)
+   * .render(_ => _.actions.add(100))
+   * component._view({}, component._init()) // output: Action<100, 'add'> and changes component state to 110
```

Comment:
- insert a line between the component definition and then the call to `component._view`
- do so for the cases above as well